subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/android/testrunner/service/TestService.kt

```diff
@@ -0,0 +1,36 @@
+package com.avito.android.testrunner.service
+
+import com.avito.android.testrunner.service.TestService.Params
+import com.avito.utils.gradle.KubernetesCredentials
+import org.gradle.api.provider.Property
+import org.gradle.api.services.BuildService
+import org.gradle.api.services.BuildServiceParameters
+
+@Suppress("UnstableApiUsage")
+public abstract class TestService : BuildService<Params>, AutoCloseable {
```

Comment:
It'd easier to see this change in another PR. It doesn't related to plugins publication

Reply:
ok, i'll extract publication logic

Reply:
Thanks